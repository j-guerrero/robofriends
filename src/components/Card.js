import React from 'react';

const Card = ({ name, email, id }) =>{

	return (
		<div className = 'tc bg-washed-green dib br3 pa1 ma2 grow'>
			<img alt='roboPic' src = {`https://robohash.org/${id}?set=set5&size=200x200`} />
			<div className="black">
				<h2> {name} </h2>
				<p> {email} </p>
			</div>
		</div>
	);
}

export default Card;