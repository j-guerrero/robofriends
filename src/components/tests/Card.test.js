import React from 'react';
import { shallow } from 'enzyme';
import Card from '../Card.js';
import {createSerializer} from 'enzyme-to-json';


expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

it('expect to render Card component', () => {
	expect(shallow(<Card />)).toMatchSnapshot();
})