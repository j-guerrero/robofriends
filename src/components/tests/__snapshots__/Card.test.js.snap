// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow"
>
  <img
    alt="roboPic"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2>
       
       
    </h2>
    <p>
       
       
    </p>
  </div>
</div>
`;
